x-pack/test/alerting_api_integration/common/fixtures/plugins/alerts/server/routes.ts

```diff
@@ -11,9 +11,97 @@ import {
   IKibanaResponse,
 } from 'kibana/server';
 import { schema } from '@kbn/config-schema';
+import { RawAlert } from '../../../../../../../plugins/alerts/server/types';
+import { TaskInstance } from '../../../../../../../plugins/task_manager/server';
+import { FixtureSetupDeps, FixtureStartDeps } from './plugin';
 
-export function defineRoutes(core: CoreSetup) {
+export function defineRoutes(
+  core: CoreSetup<FixtureStartDeps>,
+  { spaces, security }: Partial<FixtureSetupDeps>
+) {
   const router = core.http.createRouter();
+  router.put(
+    {
+      path: '/api/alerts_fixture/{id}/replace_api_key',
+      validate: {
+        params: schema.object({
+          id: schema.string(),
+        }),
+        body: schema.object({
+          spaceId: schema.maybe(schema.string()),
```

Comment:
Curious why this is needed; I think for other alerting fixtures, we take advantage of the fact that the APIs are space aware, and can get the space from the request - which means you need to invoke them with the `/s/${spaceId}` URL prefix as well.  If an explicit one is required like this, we should probably have a comment as to why, otherwise I'll be wondering 6 months from now :-)

Reply:
Because we had to exclude the `security` and `spaces` wrappers to get around the safeguards those have in place to prevent us from doing what we want to do - brute force replace the ApiKey for testing purposes :)